Src/AutoFixtureUnitTest/DataAnnotations/StringLengthAttributeRelayTest.cs

```diff
@@ -112,5 +112,29 @@ public void CreateWithConstrainedStringRequestReturnsCorrectResult(int maximum)
             Assert.Equal(expectedResult, result);
             // Teardown
         }
+
+        [Theory]
+        [InlineData(1)]
+        [InlineData(2)]
+        [InlineData(3)]
+        public void CreateWithStringRequestConstrainedbyMinimunLengthReturnsCorrectResult(int maximum)

```

Comment:
I'm not sure that I understand. The GitHub diff view seems to indicate that this test is a new test - not a modification of an existing test...